src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -236,12 +242,16 @@ export class WorkspaceSavedObjectsClientWrapper {
   }
 
   public wrapperFactory: SavedObjectsClientWrapperFactory = (wrapperOptions) => {
+    const ACLAuditor = getACLAuditor(wrapperOptions.request);
+    const clientCallAuditor = getClientCallAuditor(wrapperOptions.request);
```

Comment:
do we want to provide a flag to turn off aduit behavior?, could enabled by default.

Reply:
I was thinking that but there are too many feature flags now and I'd prefer to use the permission control feature flag. The auditor behavior won't add any impact to the UI or the server.